src/scripts/main.js

```diff
@@ -1 +1,70 @@
 'use strict';
+
+document.addEventListener('contextmenu', (e) => {
+  e.preventDefault();
+});
+
+function notification(message, isError = false) {
+  const createNotfication = document.createElement('div');
+
+  if (!isError) {
+    createNotfication.classList.add('success');
+  } else {
+    createNotfication.classList.add('error');
+  }
+
+  createNotfication.setAttribute('data-qa', 'notification');
+  createNotfication.textContent = message;
+
+  document.body.appendChild(createNotfication);
+}
+
+// Перший проміс
+
+const firstPromise = new Promise((resolve, reject) => {
+  const timeOut = setTimeout(() => {
+    resolve('First promise was rejected');
```

Comment:
The message in `resolve('First promise was rejected')` seems incorrect. Since this is a resolve call, it should indicate success, not rejection. Consider changing it to something like 'First promise was resolved after timeout'.